test/conformance/kernel/urKernelSetArgPointer.cpp

```diff
@@ -62,20 +82,26 @@ TEST_P(urKernelSetArgPointerTest, SuccessShared) {
     ASSERT_NE(allocation, nullptr);
 
     ASSERT_SUCCESS(urKernelSetArgPointer(kernel, 0, allocation));
+    ASSERT_SUCCESS(urKernelSetArgValue(kernel, 1, sizeof(data), &data));
+    ASSERT_SUCCESS(urEnqueueUSMPrefetch(queue, allocation, allocation_size,
+                                        UR_USM_MIGRATION_FLAG_DEFAULT, 0,
+                                        nullptr, nullptr));
+    Launch1DRange(array_size);
+    ValidateAllocation();
 }
 
 struct urKernelSetArgPointerNegativeTest : urKernelSetArgPointerTest {
     // Get any valid allocation we can to test validation of the other parameters.
     void SetUpAllocation() {
-        bool host_supported = false;
+        ur_device_usm_access_capability_flags_t host_supported = false;
```

Comment:
`= false` might compile but it should really be `0`. same goes for all the other instances of this check in this file.